WalletWasabi/KeyManagement/KeyManager.cs

```
@@ -298,6 +298,42 @@ public static KeyManager FromFile(string filePath)
 			return km;
 		}
 
+		public static bool TryGetEncryptedSecretFromFile(string filePath, out BitcoinEncryptedSecretNoEC encryptedSecret)
```

Functions named with Try should contains a try/catch and return false on exception or unsuccess. Add try/catch

---

Maybe we could add the exception to the log file and swallow it.

---

That's not the TryXXX pattern. https://stackoverflow.com/a/182515/2061103

The TryXXX pattern can throw exceptions.